game_quiz.py

```diff
@@ -0,0 +1,56 @@
+"""
+Welcome to Simple python quiz game
+"""
+import sys
+
```

Comment:
Line break should be of 2 lines after imports, any should be started after 2 blank lines after the imports
```suggestion
import sys


class Question
```